sdk/dotnet/S3/Inputs/BucketWebsiteGetArgs.cs

```diff
@@ -35,7 +35,7 @@ public sealed class BucketWebsiteGetArgs : Pulumi.ResourceArgs
         /// describing redirect behavior and when redirects are applied.
         /// </summary>
         [Input("routingRules")]
-        public InputUnion<string, ImmutableArray<string>>? RoutingRules { get; set; }
+        public InputUnion<string, ImmutableArray<Input<string>>>? RoutingRules { get; set; }
```

Comment:
We can probably live with this. But the current .NET SDK  can't equate/autoconvert between these two things as in:

```
        ImmutableArray<string> a;
        ImmutableArray<Input<string>> b;

        b = a;

        // MyStack.cs(15,13): error CS0029: Cannot implicitly convert type 'System.Collections.Immutable.ImmutableArray<string>' to 'System.Collections.Immutable.ImmutableArray<Pulumi.Input<string>>' [/Users/anton/tmp/play-cs-pulumi/play-cs-pulumi.csproj]
```